### PR TITLE
Problem3 - Wrote solutions for Problem3

### PR DESCRIPTION
**Pull request by farzanashampa7**

### CODE REVIEW

File: Mentorship_Problems/Problem-3.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+// nums = [1]
+// Output: 1
+
+const findSingleDigitsInArray = (nums) => {
+  let sortedNums = nums.sort();
+  for (let i = 0; i < sortedNums.length; i++) {
+    if (
+      sortedNums[i] !== sortedNums[i + 1] &&
+      sortedNums[i - 1] !== sortedNums[i]
```
Comment (Samiul-TheSoccerFan) [inline Mentorship_Problems/Problem-3.js:61]: when `i` is `0`, `sortedNums[i - 1]` becomes `sortedNums[-1]`. We definitely do not want to access a negative index.

File: Mentorship_Problems/Problem-3.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+//     Input: nums1 = [1,2,2,1], nums2 = [2,2]
+//     Output: [2]
+
+//     Input: nums1 = [4,9,5], nums2 = [9,4,9,8,4]
+//     Output: [9,4]
+//     Explanation: [4,9] is also accepted.
+//  */
+
+const findIntersectionOfTwoArrays = (nums1, nums2) => {
```
Comment (Samiul-TheSoccerFan) [inline Mentorship_Problems/Problem-3.js:80]: That works and I do want to learn more how did you reach to this solution. Can you please explain your thought process either written or verbally?

Is there a way to make it simpler? probably with `filter`?

File: Mentorship_Problems/Problem-3.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  return count;
+};
+
+console.log(findNumberOfGoodPairs([1, 2, 3]));
+
+// 2. Given a number, write a function to calculate the sum of the digits.
+//     arr = 123
+//     output = 6; 1 + 2 + 3 = 6
+
+const calculateSumOfDigits = (num) => {
```
Comment (Samiul-TheSoccerFan) [inline Mentorship_Problems/Problem-3.js:32]: This is good. Acceptable solution. Can you also think of a solution without converting the number into string?

**Review by Samiul-TheSoccerFan — COMMENTED** [review]
Left some comments

File: Mentorship_Problems/Problem-3.js (hunk 2 above)
Comment (farzanashampa7) [reply Mentorship_Problems/Problem-3.js:80]: Vaiya can I explain my thought process verbally if that's okay with you?

**Pull request merged**